plugins/neo4j-graph-widget/components/WidgetNeo4jGraph.vue

```diff
@@ -71,11 +71,34 @@
           <div class="col-12 d-flex justify-content-end align-items-end">
             <b-button type="reset" variant="danger" class="mr-2">Reset</b-button>
             <span id="disabled-wrapper">
-              <b-button type="submit" :disabled="!neo4jAppIsRunning" variant="primary">Export graph</b-button>
+              <b-button ref="submit" type="submit" :disabled="!neo4jAppIsRunning" variant="primary">Export
+                graph</b-button>
             </span>
             <b-tooltip target="disabled-wrapper" v-if="dumpButtonToolTip !== null">{{ dumpButtonToolTip }}</b-tooltip>
           </div>
         </div>
+        <b-overlay :show="showOverlay" no-wrap @shown="onShowingOverlay" @hidden="onHidingOverlay" opacity=0.95>
+          <template #overlay>
+            <div
+              ref="dialog"
+              tabindex="-1"
+              role="dialog"
+              aria-modal="false"
+              class="p-3">
+              <p>
+                A maximum of {{ neo4jDumpLimit }} documents and their related named entities will be exported.
+                <br>To export the entire graph, proceed to a <a :href="dumpDocUrl">full DB dump</a> or ask the system administrator for it.
+              </p>
+              <p>Download can take some time to start, <strong>please don't close the opened tab</strong> until then !</p>
+              <div class="d-flex align-items-center">
+                <b-button variant="danger" class="mr-3" @click="onCancellingExport">
+                  Cancel
```

Comment:
If the behavior just hide the overlay I think a back button is enough
```suggestion
                <b-button variant="outline-primary" class="mr-3" @click="onCancellingExport">
                  Back
```